### ISSUE TITLE
Loving your support examples but...

### ISSUE THREAD

**Issue by PapaDeBeau** [myflashlab/GCM-ANE#14]
I am having issues with set up. You have some good examples but they are not really in order and there are so many links to so many sites and githubs and videos and downloadable ane's and php's that it is just too hard to follow what should come frist etc...  Also I found several pages that don't have links but seem to imply there are some, like this one: http://screencast.com/t/zwngXL2v

I am hoping to plow through this fast but am feeling a little overwhelmed. On many of the others you provided a working demo that could be ran on a phone but with this one there is none and I don't even know where to start. Can you help me get up and running fairly fast? This is supposed to be very easy as your text implies but I just don't know where to start and what do to first. 

I can't get past this as well: C:\Flash Apps 2014\testing projects\Android Push GCM\Main_GCM.as, Line 260, Column 30 1046: Type was not found or was not a compile-time constant: GcmEvent.

Thanks

**Comment by myflashlab** [comment]
Dear developer(s), More than a year ago Google Deprecated GCM in favor of [FCM](https://github.com/myflashlab/Firebase-ANE). Since Google is not updating it, we didn't update it neither and have removed the product from sale. We understand that some of our customers might still be using the old GCM ANE and we'd like to help you out with the transition. Please contact us through http://www.myflashlabs.com/contact/ and ask for a discount for moving away from GCM. My colleagues should get back to you fast to let you know about how to do the transition.

Thanks,
MyFlashLabs Team
